google/cloud/storage/blob.py

```diff
@@ -2829,13 +2828,12 @@ def get_iam_policy(
         if requested_policy_version is not None:
             query_params["optionsRequestedPolicyVersion"] = requested_policy_version
 
-        info = client._connection.api_request(
-            method="GET",
-            path="%s/iam" % (self.path,),
+        info = client._get_resource(
+            "%s/iam" % (self.path,),
```

Comment:
This is more of just a style thing, but in modern Python, should we be preferring `str.format` or `f` strings if Python 2 support has been dropped?

Reply:
Once we drop Python2, I would definitely prefer f-strings.